packages/sdk/package.json

```diff
@@ -31,6 +31,7 @@
 	"author": "Sygmaprotocol Product Team",
 	"license": "LGPL-3.0-or-later",
 	"devDependencies": {
+		"@chainsafe/chainbridge-contracts": "2.1.4",
```

Comment:
We are using `@buildwithsygma/sygma-contracts` package here. Do you have specific reason to add `@chainsafe/chainbridge-contracts`?

Reply:
https://github.com/sygmaprotocol/sygma-sdk/blob/main/packages/sdk/integration/test/utils.test.ts#L3

Reply:
Note that this is dev dependency, I haven't remove prod dep^^